app/models/effective/form_input.rb

```diff
@@ -180,13 +178,10 @@ def build_feedback
 
       invalid = object.errors[name].to_sentence.presence if object.respond_to?(:errors)
       invalid ||= options[:feedback][:invalid].delete(:text).presence
-      invalid ||= [("can't be blank" if options[:input][:required]), ('must be valid' if validated?(name))].tap(&:compact!).join(' and ').presence
-      invalid ||= "can't be blank or is invalid"
+      invalid ||= [("Can't be blank" if options[:input][:required]), ('must be valid' if validated?(name))].tap(&:compact!).join(' and ').presence
+      invalid ||= "Can't be blank or is invalid"
 
-      valid = options[:feedback][:valid].delete(:text) || 'Looks good!'
-
-      content_tag(:div, invalid.html_safe, options[:feedback][:invalid]) +
-      content_tag(:div, valid.html_safe, options[:feedback][:valid])
```

Comment:
Is this removing the green text entirely? No valid feedback?

Reply:
We've never really used the positive feedback before. Sometimes I override the negative.  Maybe a better solution would be to leave the valid text if present, but set it to empty string by default?

Reply:
yeah removing the green text. the valid feedback is still there as: 1) a big green checkmark, 2) a green outline on the field. 

I don't like "Looks good!" repeated over the page because it doesn't look good, and it distracts from the fields that have actual error messages below them

Reply:
I like this change